app/Models/EventParticipation.php

```diff
@@ -62,6 +62,7 @@ public function getIsActiveQuizAllowedAttribute()
             return;
         }
 
-        return $this->event->activeQuiz->participations()->where('team_id', $this->team->id)->exists();
+        // return $this->event->activeQuiz->participations()->where('team_id', $this->team->id)->exists();
+        return $this->event->activeQuiz->participations()->with('team');
```

Comment:
Eager loading of relationships should not be the responsibilty of a accessor method inside a model!

How does this improve queries? Give me some context.